lib/features/rules_filter_creator/presentation/model/rules_filter_input_field_arguments.dart

```diff
@@ -0,0 +1,22 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+import 'package:equatable/equatable.dart';
+import 'package:flutter/material.dart';
+
+class RulesFilterInputFieldArguments with EquatableMixin {
+  final FocusNode inputRuleConditionValueFocusNode;
```

Comment:
```suggestion
  final FocusNode focusNode;
```
I think class name is enough to describe, so no need long param name